cmplibrary/src/test/java/com/sourcepoint/gdpr_cmplibrary/StoreClientTest.java

```diff
@@ -101,25 +98,60 @@ public void getAuthId() {
         storeClient.setAuthId(authId);
         assertEquals(authId,storeClient.getAuthId());
         editor.clear().commit();
-        assertFalse(sharedPreferences.contains(AUTH_ID_KEY));
+        assertFalse(sharedPreferences.contains(StoreClient.AUTH_ID_KEY));
     }
 
     @Test
     public void clearAllData() {
         storeClient.clearAllData();
-        assertFalse(sharedPreferences.contains(CONSENT_UUID_KEY));
-        assertFalse(sharedPreferences.contains(META_DATA_KEY));
-        assertFalse(sharedPreferences.contains(EU_CONSENT__KEY));
-        assertFalse(sharedPreferences.contains(AUTH_ID_KEY));
+        assertFalse(sharedPreferences.contains(StoreClient.CONSENT_UUID_KEY));
+        assertFalse(sharedPreferences.contains(StoreClient.META_DATA_KEY));
+        assertFalse(sharedPreferences.contains(StoreClient.EU_CONSENT_KEY));
+        assertFalse(sharedPreferences.contains(StoreClient.AUTH_ID_KEY));
         assertFalse(sharedPreferences.contains(IAB_CONSENT_CONSENT_STRING));
     }
 
     @Test
     public void clearInternalData() {
         storeClient.clearInternalData();
-        assertFalse(sharedPreferences.contains(CONSENT_UUID_KEY));
-        assertFalse(sharedPreferences.contains(META_DATA_KEY));
-        assertFalse(sharedPreferences.contains(EU_CONSENT__KEY));
-        assertFalse(sharedPreferences.contains(AUTH_ID_KEY));
+        assertFalse(sharedPreferences.contains(StoreClient.CONSENT_UUID_KEY));
+        assertFalse(sharedPreferences.contains(StoreClient.META_DATA_KEY));
+        assertFalse(sharedPreferences.contains(StoreClient.EU_CONSENT_KEY));
+        assertFalse(sharedPreferences.contains(StoreClient.AUTH_ID_KEY));
     }
+
+
+    @Test
+    public void setTCDataAndcheckPrimitives() {
+        editor.clear().commit();
+
+        HashMap<String, Object> tcData = new HashMap<>();
+        tcData.put("IABTCF_String", "foo");
+        tcData.put("IABTCF_Integer" , 100);
+        storeClient.setTCData(tcData);
+
+        Map map = sharedPreferences.getAll();
+
+        assertEquals("foo", map.get("IABTCF_String"));
+        assertEquals(100 , map.get("IABTCF_Integer"));
+    }
+
+    @Test
+    public void getTCDataAndCheckPrimitives() {
```

Comment:
There is no need for this test since the one before does the job already

Reply:
@leodoin This is test method for getTcData , so we should have this test case.